frontend/src/components/ThemedInput/ThemedNumberInput.css

```diff
@@ -7,5 +7,6 @@
 
 /* hide arrows in firefox */
```

Comment:
super duper nitpicky but can this comment go next to the `appearance: textfield;` line